Controller/ContentTypeController.php

```diff
@@ -225,7 +223,7 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
             'modifier' => $this->userService->loadUser($contentType->modifierId),
             'delete_form' => $deleteForm->createView(),
             'can_edit' => $this->isGranted(new Attribute('class', 'update')),
-            'can_delete' => ($this->isGranted(new Attribute('class', 'delete')) && $contentCount == 0),
+            'can_delete' => ($this->isGranted(new Attribute('class', 'delete')) && !$this->contentTypeService->isContentTypeUsed($contentType)),
```

Comment:
another nitpick and not related entirely with this pr but as more high level... doesn't ```class``` sounds a bit legacy? :wink: i mean, maybe we could think in change that to "contentType" too...